common.sh

```diff
@@ -15,24 +17,59 @@ OCS_BASE_PATH=`absPath ${OCS_BASE_PATH:-"$DEV_BASE_PATH/ocs"}`
 JRE_PATH=`absPath ${JRE_PATH:-"$DEV_BASE_PATH/jres8"}`
 
 
+# ********************************
+# ***** VARIABLES AND ARRAYS *****
+# ********************************
+# Executables should add their configuration parameters to these arrays.
+# Single value variables NAME=VALUE should add NAME to VARIABLES.
+# Arrays NAME=(VALUE1 VALUE2 ...) should add NAME to ARRAYS.
+declare -a VARIABLES
+declare -a ARRAYS
+
+
+# Set up the ssh ids.
+# We use exit here instead of return as we want to terminate if this fails
+# without requiring the calling code to handle.
+function sshIds() {
+    logHeader "SSH ID setup" no
+    
+    eval "$BASE_DIR/keysetup.exp"
+    if [[ $? -ne 0 ]]; then
+	logError "Could not set up SSH IDs."
+	exit 1
+    fi
+    logInfo "SSH ID setup complete."
+    return 0
+}
+
+# setjdk always returns 0, even for failure.
+function javaVersion() {
+    logHeader "Setting JDK version"
+    setjdk 1.8.0_73
```

Comment:
Yes, we will have to update this, unfortunately. I am hoping the message in the build feedback will at least remind us that it needs to be done, but am open to better suggestions.